src/modules/robot/arm_solutions/RotatableDeltaSolution.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RotatableDeltaSolution_H
+#define RotatableDeltaSolution_H
+#include "libs/Module.h"
+#include "BaseSolution.h"
+
+class Config;
+
+class RotatableDeltaSolution : public BaseSolution {
+    public:
+        RotatableDeltaSolution(Config*);
+        void cartesian_to_actuator( float[], float[] );
+        void actuator_to_cartesian( float[], float[] );
+
+        bool set_optional(const arm_options_t& options);
+        bool get_optional(arm_options_t& options);
+
+    private:
+        void init();
+        int delta_calcAngleYZ(float x0, float y0, float z0, float &theta);
+        int delta_calcForward(float theta1, float theta2, float theta3, float &x0, float &y0, float &z0);
+
+        float delta_e;			// End effector length
+        float delta_f;			// Base length
+        float delta_re;			// Carbon rod length
+        float delta_rf;			// Servo horn length
+        float delta_z_offset ;		// Distance from delta 8mm rod/pulley to table/bed
+			       		// NOTE: For OpenPnP, set the zero to be about 25mm above the bed
+
+        float delta_ee_offs;		// Ball joint plane to bottom of end effector surface
+        float tool_offset;		// Distance between end effector ball joint plane and tip of tool
+        float z_calc_offset;
+
+        float z_home_offs;		// This is calculated from the above angle, after applying forward
+        				// kinematics, and adding the Z calc offset to it.
+
+        float xyz_full_steps_per_rotation;	// stepper motor steps per 360 full rotation
```

Comment:
I have high lighted in green the method that uses the variable, and if red if it should have been removed (and now has been). 
> On 11/06/2015, at 1:57 pm, Jim Morris <notifications@github.com> wrote:
> 
> In src/modules/robot/arm_solutions/RotatableDeltaSolution.h <https://github.com/Smoothieware/Smoothieware/pull/673#discussion_r32184465>:
> 
> > +
> > +        delta_calcAngleYZ float delta_e;			// End effector length
> > +        delta_calcAngleYZ float delta_f;			// Base length
> > +        delta_calcAngleYZ float delta_re;			// Carbon rod length
> > +        delta_calcAngleYZ float delta_rf;			// Servo horn length
> > +        init float delta_z_offset ;		// Distance from delta 8mm rod/pulley to table/bed
> > +			       		// NOTE: For OpenPnP, set the zero to be about 25mm above the bed
> > +
> > +        init float delta_ee_offs;		// Ball joint plane to bottom of end effector surface
> > +        init float tool_offset;		// Distance between end effector ball joint plane and tip of tool
> > +        init & cartesian_to_actuator float z_calc_offset;
> > +
> > +        CAN GO NOT NEEDED!!! float z_home_offs;		// This is calculated from the above angle, after applying forward
> > +        				// kinematics, and adding the Z calc offset to it.
> > +
> > +        CAN GO NOT NEEDED!!! float xyz_full_steps_per_rotation;	// stepper motor steps per 360 full rotation
> still unused
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Smoothieware/Smoothieware/pull/673/files#r32184465>.
>